spec/models/chargeback_vm_spec.rb

```diff
@@ -905,19 +905,24 @@ def result_row_for_vm(vm)
           FactoryGirl.create(:metric_rollup_vm_hr, :timestamp => report_run_time - 1.day - 17.hours,
                              :parent_host_id => @host1.id, :parent_ems_cluster_id => @ems_cluster.id,
                              :parent_ems_id => ems.id, :parent_storage_id => @storage.id,
-                             :resource => @vm1)
+                             :resource => @vm)
         end
         let(:consumption) { Chargeback::ConsumptionWithRollups.new([metric_rollup], nil, nil) }
 
         before do
           @storage.tag_with([classification_1.tag.name, classification_2.tag.name], :ns => '*')
           ChargebackRate.set_assignments(:storage, [rate_assignment_options_1, rate_assignment_options_2])
+          @vm = FactoryGirl.create(:vm_vmware, :name => "test_vm_1", :evm_owner => admin, :ems_ref => "ems_ref", :created_on => month_beginning)
         end
 
         it "return only one chargeback rate according to tag name of Vm" do
+          skip('this feature needs to be added to new chargeback') if Settings.new_chargeback
+
           [rate_assignment_options_1, rate_assignment_options_2].each do |rate_assignment|
             metric_rollup.tag_names = rate_assignment[:tag].first.tag.send(:name_path)
-            uniq_rates = chargeback_vm.get(consumption)
+            @vm.tag_with(["/managed/#{metric_rollup.tag_names}"], :ns => '*')
```

Comment:
Do we have a test where a rollup is tagged with one set of tags and the resource/vm is tagged with an additional tag and we confirm that the additional tag is used for running the chargeback report?

Reply:
Very good point 👍  because we don't and  I found bug during writing this test.